app/views/summary/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>
+<small>Summary for:</small><br>
+  <%= @post.title %>
+</h1>
+<div class="row">
+  <div class="col-md-12">
+    <%= form_for [@summary], url: topic_post_summary_index_path(@topic, @post, @summary) do |f| %>
```

Comment:
Oh, and here I had to explicitly specify the url because of the whole singular resource. Plus, I had to use the `_index` version ... `rake routes` didn't show a named path created for the `:create` action ... I'm not sure why.